src/Internal/Transition.elm

```diff
@@ -0,0 +1,69 @@
+module Internal.Transition exposing
+    ( Config
+    , Property(..)
+    , PropertyShorthand
+    , Transition(..)
+    , all
+    , render
+    , toAttr
+    )
+
+import Html exposing (Attribute)
+import Html.Attributes
+import Internal.Animation
+    exposing
+        ( Millis
+        , Option
```

Comment:
I considered creating a `Internal.Option` and moving all the common option stuff there, thoughts?

Reply:
Interesting, I think in this case it would be good to define the options for `Transition` within this module.

`Ease` and `Delay` are the same as in  `Internal.Animation`, but a transition doesn't have the concept of `Iteration`. It might also 

If you feel like it I think `Ease` could be good to extract into a shared module as css animations and transitions both share eases.

Does that make sense?